### PR TITLE
Balancer: Fee collector final deploy

### PR DESCRIPTION
**Pull request by facuspagnuolo**

### CODE REVIEW

File: tasks/2023053100-balancer-fee-collector-updates-mainnet/input.ts [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+        txCostLimit: 0,
+        gasToken: tokens.USDC,
+        permissiveMode: true,
+        relayers: [BOT],
+      },
+      timeLockConfig: {
+        delay: 14 * DAY,
+        nextExecutionTimestamp: 1686304800,
+      },
+      tokenIndexConfig: defaultTokenIndexConfig,
```
Comment (dmf7z) [inline tasks/2023053100-balancer-fee-collector-updates-mainnet/input.ts:161]: I think we should allow it only for USDC and BAL. This way the bot does not need to know the configuration offchain and can iterate over those two tokens.

File: tasks/2023053101-balancer-fee-collector-updates-optimism/input.ts [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+        txCostLimit: 0,
+        gasToken: tokens.USDC,
+        permissiveMode: true,
+        relayers: [BOT],
+      },
+      timeLockConfig: {
+        delay: 14 * DAY,
+        nextExecutionTimestamp: 1686304800,
+      },
+      tokenIndexConfig: defaultTokenIndexConfig,
```
Comment (dmf7z) [inline tasks/2023053101-balancer-fee-collector-updates-optimism/input.ts:163]: same here, USDC and BAL

File: tasks/2023053102-balancer-fee-collector-updates-l2/input.arbitrum.ts [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+          gasToken: tokens.USDC,
+          permissiveMode: true,
+          relayers: [BOT],
+        },
+        timeLockConfig: {
+          delay: 14 * DAY,
+          nextExecutionTimestamp: 1686276000,
+        },
+        tokenIndexConfig: defaultTokenIndexConfig,
+        tokenThresholdConfig: defaultTokenThresholdConfig,
```
Comment (dmf7z) [inline tasks/2023053102-balancer-fee-collector-updates-l2/input.arbitrum.ts:151]: No threshold so we assure execution

File: tasks/2023053102-balancer-fee-collector-updates-l2/input.gnosis.ts [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+          gasToken: tokens.USDC,
+          permissiveMode: true,
+          relayers: [BOT],
+        },
+        timeLockConfig: {
+          delay: 14 * DAY,
+          nextExecutionTimestamp: 1686276000,
+        },
+        tokenIndexConfig: defaultTokenIndexConfig,
+        tokenThresholdConfig: defaultTokenThresholdConfig,
```
Comment (dmf7z) [inline tasks/2023053102-balancer-fee-collector-updates-l2/input.gnosis.ts:151]: No threshold so we assure execution

File: tasks/2023053102-balancer-fee-collector-updates-l2/input.polygon.ts [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+          gasToken: tokens.USDC,
+          permissiveMode: true,
+          relayers: [BOT],
+        },
+        timeLockConfig: {
+          delay: 14 * DAY,
+          nextExecutionTimestamp: 1686276000,
+        },
+        tokenIndexConfig: defaultTokenIndexConfig,
+        tokenThresholdConfig: defaultTokenThresholdConfig,
```
Comment (dmf7z) [inline tasks/2023053102-balancer-fee-collector-updates-l2/input.polygon.ts:151]: No threshold so we assure execution

File: tasks/2023053102-balancer-fee-collector-updates-l2/input.gnosis.ts [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+import { BOT } from '../../constants/mimic'
+import * as tokens from '../../constants/tokens/gnosis'
+import Task from '../../src/task'
+
+const SmartVault = new Task('2023050301-balancer-fee-collector-l2-v2')
+const PermissionsManager = new Task('2023050301-balancer-fee-collector-l2-v2')
+
+/* eslint-disable no-secrets/no-secrets */
+
+const OWNER = '0x09Df1626110803C7b3b07085Ef1E053494155089'
```
Comment (dmf7z) [inline tasks/2023053102-balancer-fee-collector-updates-l2/input.gnosis.ts:13]: `0x14969B55a675d13a1700F71A37511bc22D90155a`

**Review by dmf7z — CHANGES_REQUESTED** [review]

File: tasks/2023053100-balancer-fee-collector-updates-mainnet/input.ts (hunk 1 above)
Comment (facuspagnuolo) [reply tasks/2023053100-balancer-fee-collector-updates-mainnet/input.ts:161]: Nice catch! These type of things will make the bot run much faster, we should bear it in mind for other cases as well

File: tasks/2023053102-balancer-fee-collector-updates-l2/input.arbitrum.ts (hunk 3 above)
Comment (facuspagnuolo) [reply tasks/2023053102-balancer-fee-collector-updates-l2/input.arbitrum.ts:151]: Are you sure? The config doc does mention a threshold for the bridges

**Review by dmf7z — APPROVED** [review]

**Pull request merged**